app/Controllers/LibriController.php

```
@@ -3212,6 +3237,14 @@ public function exportCsv(Request $request, Response $response, mysqli $db): Res
                 array_map('intval', explode(',', $idsParam)),
                 static fn (int $id): bool => $id > 0
             )));
+            // The selected-export path caps at 1000 IDs. Don't truncate in
+            // silence: log it so a large multi-page selection that loses rows
+            // is at least diagnosable (the UI accumulates IDs across pages).
+            if (count($selectedIds) > 1000) {
+                SecureLogger::error(__('Export selezione troncato a 1000 libri'), [
+                    'requested' => count($selectedIds),
+                ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\SecureLogger' in method 'exportCsv'. (undefined)

(StaticAccess)